po/de.po

```diff
@@ -11182,14 +11170,11 @@ msgid "Import Account Assistant"
 msgstr "Konten-Import-Assistent"
 
 #: gnucash/gtkbuilder/assistant-csv-account-import.glade:66
-#, fuzzy
 #| msgid ""
 #| "\n"
 #| "Enter file name and location for the Import...\n"
 msgid "Enter file name and location for the Import…"
-msgstr ""
-"\n"
-"Bestimmen Sie Ort und Namen der Datei für den Import...\n"
+msgstr "Bestimmen Sie Ort und Namen der Datei für den Import..."
```

Comment:
We switched to hellipsis `…`  (with one exception, which I had not seen).
AFAIK weblate can mass change them.